File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.c
#include <stdio.h>

int main(void) { 
   int a, b, r;
   scanf("%d %d", &a, &b);

   int new_a = a, new_b = b;
   while(1){
      r = a % b;
      if(r == 0){
         printf("%d\n", b);
         break;
      }
      else{
         a = b;
         b = r;
      }
   }

   printf("%d", (new_a * new_b) / b);
   
   return 0;
}

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
a, b = map(int, input().split())

new_a, new_b = a, b
while(True):
  r = new_a % new_b
  if r == 0:
    print(new_b)
    break
  else:
    new_a = new_b
    new_b = r

print(a * b // new_b)